e2e/cypress/e2e/organization/organizations.cy.ts

```diff
@@ -21,6 +21,15 @@ describe('organizations', () => {
         });
     });
 
+    // afterEach(() => {
+    //   cy.visit(orgPath);
+    //   apiAuth()
+    //     .as('api')
+    //     .then((api) => {
+    //       renameOrg(api, orgNameOnCreation);
+    //     });
+    // });
+
```

Comment:
should we not delete unused code?

Reply:
you're file is outdated. This is the current one: https://github.com/zitadel/zitadel/blob/2c06bcecf5e630f4fcd0492402594cf0b728080e/e2e/cypress/e2e/organization/organizations.cy.ts